app/index.js

```diff
@@ -39,7 +39,7 @@ module.exports = generators.NamedBase.extend({
     }, {
       type: 'input',
       name: 'userName',
-      message: 'What is your name? (Used for the package.json.)',
+      message: 'What is your name? (Used for the package.json. Capitalize the first character)',
```

Comment:
@royxue That value is transformed and [used three ways](https://github.com/derekpeterson/generator-react-flask/blob/master/app/index.js#L57): kebab-case (package.json), snake_case (Python), and PascalCase (React component). Since we override the case that they specify, it's not a big deal.